airbyte-migration/src/main/java/io/airbyte/migrate/migrations/MigrationV0_14_0.java

```diff
@@ -86,6 +86,7 @@ public void migrate(Map<ResourceId, Stream<JsonNode>> inputData, Map<ResourceId,
     DESTINATION_CONNECTION,
     STANDARD_SYNC,
     STANDARD_SYNC_SCHEDULE,
+    STANDARD_SYNC_OPERATION,
```

Comment:
you don't need to modify the MivrationV0_14_0.java file as this configuration was not introduced in airbyte v.0.14.0 but in 0.25.0

Reply:
@ChristopheDuong I did it because the test `testConfigsOfLastMigrationMatchSource` in `MigrationCurrentSchemaTest` relies on this enum for assertions

https://github.com/airbytehq/airbyte/blob/master/airbyte-migration/src/test/java/io/airbyte/migrate/MigrationCurrentSchemaTest.java#L46

Reply:
I see, should we migrate the enums out of the specific migration class file for 0.14.0 then as it defines what files should be checked in general by all migrations?